### PR TITLE
Added in org ID and name to intercom data

### PR DESCRIPTION
**Pull request by beweinreich**
We will be syncing org ID across Intercom, Mixpanel, and Salesforce. This pushes org ID and name to intercom fields.

### CODE REVIEW

File: src/components/intercom/index.tsx [hunk 1]
```diff
@@ -9,7 +9,9 @@ export function IntercomDensity({ user }) {
     intercomUser = {
       user_id: user.data.id,
       email: user.data.email,
-      name: user.data.fullName
+      name: user.data.fullName,
+      org_id: user.data.organization.id,
```
Comment (guscost) [inline src/components/intercom/index.tsx]: Could we use `organization_name` as it's the convention in most other places, or does this have to match a convention among the user analytics tools?

Comment (beweinreich) [reply src/components/intercom/index.tsx]: yeah, its already org_name in the other tools, so I'm just being consistent.

Comment (beweinreich) [reply src/components/intercom/index.tsx]: sorry— to clarify, it's already org_name in Mixpanel and Intercom, so I want to keep it consistent. 

yeah yeah, i know im usually the big bad wolf when it comes to abbreviations in this stuff, but a lot of these fields in these third party systems have max key sizes :(

Comment (beweinreich) [reply src/components/intercom/index.tsx]: waaaiiiit. i lied! mixpanel actually has `organization_id` and `organization`. ugh, i wish I had done `organization_name`. Just checked MP and SFDC and they look like they can handle that length. 

okay, changing!

**Pull request merged**